.mega-linter.yml

```diff
@@ -22,7 +22,7 @@ ENABLE_LINTERS:
   - ENV_DOTENV_LINTER
   - YAML_YAMLLINT
   - JSON_JSONLINT
-  - ACTION_ACTIONLINT
+# - ACTION_ACTIONLINT
```

Comment:
Not a blocker: can you open an issue to add actionlint back in and reference it in this pr. Also consider a comment explaining why actionlint was disabled , with the intention of having a separate pr to resolve the bash linting issues that is part of the the run action.

Reply:
yes 👍 quick note; I fixed an issue related to actionlinter here; https://github.com/microsoft/symphony/pull/234